org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLHover.java

```diff
@@ -188,7 +215,7 @@ private Hover getAttrValueHover(HoverRequest hoverRequest, Range attrRange) {
 	 * @param hoverRequest the hover request.
 	 * @return the aggregated LSP hover from the value list.
 	 */
-	private static Hover creatHover(List<String> contentValues, HoverRequest hoverRequest) {
+	private static Hover createHover(List<String> contentValues, HoverRequest hoverRequest) {

```

Comment:
This was just a typo I found, not related to the objective of this PR.

Reply:
Please change that is this same PR. Thanks for having see that.

Reply:
Ok so the rename is okay?